web/modules/custom/myroute/src/Controller/MyrouteController.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace Drupal\myroute\Controller;
+
+use Drupal\Core\Controller\ControllerBase;
+use Drupal\Core\Session\AccountInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * Defines a controller for the Myroute module.
+ */
+class MyrouteController extends ControllerBase {
+
+  /**
+   * The account object for the currently logged-in user.
```

Comment:
It can't be for logged out user?

Reply:
Just write User account service or something.

Reply:
Changed as per the suggestion.